projects/docs/docs/building-at-scale/caching.md

```diff
@@ -16,28 +16,61 @@ we do caching at the module level. That means developers can use Xcode and its b
 ### Warming the cache
 
 Warming the cache is the process of building, hashing, and storing cacheable targets in the cache.
-We recommend setting up a continuous integration pipeline that runs on every master commit and executes the following command:
+We recommend setting up a continuous integration pipeline that runs on every master commit and executes the [cache](commands/cache.md) command:
 
-```
+```bash
 tuist cache warm
 ```
 
 To warm the cache of only specific targets and their dependencies, you can run:
 
-```
+```bash
 tuist cache warm FrameworkA FrameworkB
 ```
 
-One the cache is warmed, you can use the [generate](commands/generate.md) command to generate a project replacing direct and transitive dependencies with artifacts from the cache.
+To warm the cache of only targets not defined in your project or workspace (for example, external dependencies), you can run:
+
+```bash
+tuist cache warm --dependencies-only
+```
```

Comment:
With the layout of this page it may make sense to introduce this in the `### External dependencies and cache` section

Reply:
I would keep it here. Dependencies is not strictly related to external dependencies. You can still do `tuist cache warm MyTarget --dependencies-only`, which would be to cache dependencies of MyTArget, including the internal ones